### PR TITLE
fix(whatsapp): update location URL placeholder and regex to wa.me format (#30118)

### LINKED ISSUE #30118: WhatsApp LIEU (location) generates wrong URL (/send and + included)
Description:
In the configuration (setup) page, when adding a WhatsApp LIEU (Location), the generated information is incorrect. It displays a send-style URL, but the actual WhatsApp URL should be in the format:

https://wa.me/4712345678

It should not include a + sign or /send.

Expected:
WhatsApp LIEU (Location) should use a clean wa.me URL without + and without /send.

Actual:
The UI shows a URL like a send link instead of the correct WhatsApp link format. And it will not work.

Example:
Correct: https://wa.me/4712345678
Incorrect: https://wa.me/send?phone=4712345678 (or similar)

### PR DESCRIPTION
**Pull request by Dewin**
## What does this PR do?

Updates the WhatsApp location input placeholder and validation regex in `packages/app-store/whatsapp/config.json` to enforce the canonical `https://wa.me/<number>` URL structure, removing the legacy `/send` path and preventing validation issues.

- Fixes #30118

### Root Cause
`organizerInputPlaceholder` specified `https://wa.me/send?phone=1234567890`, which was misleading organizers to use send-style URLs that fail on mobile devices and did not match clean wa.me requirements.

### Changes
- Updated `organizerInputPlaceholder` to `https://wa.me/1234567890`
- Updated `urlRegExp` to `^http(s)?:\/\/(www\.)?wa.me\/[0-9]+`
- Added unit tests in `packages/app-store/whatsapp/test/whatsapp-url.test.ts`

## Mandatory Tasks
- [x] I have self-reviewed the code.
- [x] I have updated the developer docs if needed (N/A).
- [x] I confirm automated tests are in place.

### CODE REVIEW

File: packages/app-store/whatsapp/config.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
       "type": "integrations:whatsapp_video",
       "label": "WhatsApp",
       "linkType": "static",
-      "organizerInputPlaceholder": "https://wa.me/send?phone=1234567890",
-      "urlRegExp": "^http(s)?:\\/\\/(www\\.)?wa.me\\/[a-zA-Z0-9]*"
+      "organizerInputPlaceholder": "https://wa.me/1234567890",
+      "urlRegExp": "^http(s)?:\\/\\/(www\\.)?wa.me\\/[0-9]+"
```
[the remark this answers is not in the document]
Comment (Dewin) [reply packages/app-store/whatsapp/config.json]: Addressed: escaped the dot in \wa\\.me\ and added the \$\ end anchor in \urlRegExp\ to strictly reject malformed hosts, suffix characters, and trailing paths. Added unit regression tests covering all edge cases.

**Comment by Dewin** [comment]
Still active and ready for review! Automated checks and CodeRabbit review have passed.
